js_modules/dagit/packages/core/src/app/AppTopNav.test.tsx

```diff
@@ -1,9 +1,9 @@
 import {act, render, screen, waitFor, within} from '@testing-library/react';
 import * as React from 'react';
 
+import {InstigationStatus} from '../graphql/graphql';
 import {DeploymentStatusProvider, DeploymentStatusType} from '../instance/DeploymentStatusProvider';
 import {TestProvider} from '../testing/TestProvider';
-import {InstigationStatus} from '../types/globalTypes';
```

Comment:
Also no more `../types/globalTypes`. Global types now live alongside other types in `../graphql/graphql`.